src/assets/js/map.js

```diff
@@ -83,6 +83,13 @@ export class Map {
 
       // Convert the coordinates string to a GeoJSON object
       const geometry = parse(geometryWkt)
+
+      // if the geometry is invalid, log an error and continue
+      if (!geometry) {
+        console.error('Invalid WKT geometry format', geometryWkt)
+        return null
```

Comment:
This inline function return type is void, so it doesn't need to return anything. Otherwise looks OK.

Reply:
should be fine either way, thanks for raising

Reply:
I'll remove it because I need to rebase anyways